cyral/resource_cyral_sidecar_test.go

```diff
@@ -105,65 +105,71 @@ func TestAccSidecarResource(t *testing.T) {
 	testUpdateConfigBypassNever, testUpdateFuncBypassNever := setupSidecarTest(bypassNeverSidecarConfig)
 	testUpdateConfigBypassAlways, testUpdateFuncBypassAlways := setupSidecarTest(bypassAlwaysSidecarConfig)
 
-	resource.ParallelTest(t, resource.TestCase{
-		ProviderFactories: providerFactories,
-		Steps: []resource.TestStep{
-			{
-				Config: testConfig,
-				Check:  testFunc,
-			},
-			{
-				Config: testUpdateConfigDocker,
-				Check:  testUpdateFuncDocker,
-			},
-			{
-				Config: testUpdateConfigHelm,
-				Check:  testUpdateFuncHelm,
-			},
-			{
-				Config: testUpdateConfigTF,
-				Check:  testUpdateFuncTF,
-			},
-			{
-				Config: testUpdateConfigSingleContainer,
-				Check:  testUpdateFuncSingleContainer,
-			},
-			{
-				Config: testUpdateConfigLinux,
-				Check:  testUpdateFuncLinux,
-			},
-			{
-				Config: testUpdateConfigBypassNever,
-				Check:  testUpdateFuncBypassNever,
-			},
-			{
-				Config: testUpdateConfigBypassAlways,
-				Check:  testUpdateFuncBypassAlways,
-			},
-			{
-				ImportState:       true,
-				ImportStateVerify: true,
-				ResourceName:      "cyral_sidecar.test_sidecar",
+	resource.ParallelTest(
```

Comment:
Just out of curiosity, what is the formatting tool you use? In my case I use the Go VSCode extension, which is based on the `gofmt`, I notice that it does not conflict with this formatting style, but at the same time it does not enforce that.

Reply:
As a friendly suggestion, separate actual changes from linter execution in different commits to make it easier to review.

Reply:
I use whatever Goland (IntelliJ IDEA Ultimate) has set by default for Go. It runs `gofmt` when I reformat code, however I think I have some additional rules such as chomp down on long lines that `gofmt` ignores.

Reply:
@wcmjunior easier said than done. My IDE automatically reformats on save. I would prefer not to turn that off generally, but I can for this project.

Reply:
I'm on the same boat as Victor as I also use the standard VSCode extension: no complains on the format, but it will also not produce the same output. I should have mention that this is not necessary for this PR as it does not make sense, but next PRs it would be helpful.